moveit_core/kinematics_base/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ set_target_properties(${MOVEIT_LIB_NAME} PROPERTIES VERSION "${${PROJECT_NAME}_V
 
 # Avoid warnings about deprecated members of KinematicsBase when building KinematicsBase itself.
 # TODO: remove when deperecated variables (tip_frame_, search_discretization_) are finally removed.
-# target_compile_options(${MOVEIT_LIB_NAME} PRIVATE -Wno-deprecated-declarations)
+target_compile_options(${MOVEIT_LIB_NAME} PRIVATE -Wno-deprecated-declarations)
```

Comment:
Didn't we remove all deprecated code? If not I think we should rather remove those than suppressing the warnings.

Reply:
The two variables are still used [here](https://github.com/ros-planning/moveit2/blob/ec878c50d727d35149efb11cc82fc00d22bef2a8/moveit_core/kinematics_base/src/kinematics_base.cpp#L65) and [here](https://github.com/ros-planning/moveit2/blob/ec878c50d727d35149efb11cc82fc00d22bef2a8/moveit_core/kinematics_base/src/kinematics_base.cpp#L84). I'm not sure if they are still used with some legacy IK plugin.

Reply:
They are still there but don't appear to really be used. Let's remove them

Reply:
I have removed these deprecated variables used in kinematics_base.